posix.mak

```diff
@@ -358,7 +358,7 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 # The mktemp business is needed so .o files don't clash in concurrent unittesting.
 %.test : %.d $(LIB)
 	T=`mktemp -d /tmp/.dmd-run-test.XXXXXX` && \
-	  $(DMD) -od$$T $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) -cov -run $< && \
+	  $(DMD) -od$$T $(DFLAGS) -g -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) -cov -run $< && \
```

Comment:
Without `-g` I got gibberish stack traces on Travis, but yeah as soon as the other problems are resolved, I can remove that one ;-)